index.html

```diff
@@ -2723,66 +2709,63 @@ <h2>Protocol Bindings</h2>
                 as identified by the Internet Media Type can also be
                 shared across these generic protocol stacks.
             </p>
-            <p> In some cases, where no aspects can be shared, the <a>Protocol
-                    Binding</a> is more comparable to a platform-specific
-                driver that is selected and configured through the
-                communication metadata in similar way as above.
-            </p>
-            <p> See <a
-                    href="https://w3c.github.io/wot-binding-templates/">WoT
-                    Binding Templates</a> for details.
+            <p>
+                See [[?wot-binding-templates]] for details.
             </p>
         </section>
         <section>
             <h2>System API</h2>
-            <p> The implementation of a <a>Thing</a> may access local
-                hardware or system services (e.g. storage) through
-                proprietary APIs or other means. This block is out of
-                scope of WoT standardization.
-            </p>
-            <p> However, a given implementation of a <a>WoT Runtime</a>
+            <p>
+                A given implementation of a <a>WoT Runtime</a>
                 may provide local hardware or system services to
-                application scripts through a <a>Thing</a>
-                abstraction, as if they were accessible over a network
-                protocol. In this case the implementation should be
-                optimized to avoid the overhead that would be involved
-                in a literal implementation of a network protocol while
-                maintaining a consistent <a>WoT Interface</a>.
-            </p>
-            <p> A device may also be physically external to a <a>Servient</a>,
-                but connected via a proprietary protocols or a protocol
-                not eligible for direct support as part of the WoT. In
-                this case, the implemented runtime environment may
+                behavior implementations through the <a>Thing</a>
+                abstraction, as if they were accessible over a communication
+                protocol. In this case, the <a>WoT Runtime</a> should
+                enable the behavior implementation to instantiate <a>Consumed Things</a>
+                that internally interface with the system instead of the protocol stack.
+                This can be done by listing such system Things in the discovery results.
```

Comment:
What is the implication of being a "system Thing"? IMO it means that there does not exist a TD but only an object in the runtime. 
"Discovery" probably means local introspection.

Reply:
Rephrased: in the results of the discovery mechanism provided by the application-facing <a>WoT Runtime</a> API